jwt.go

```diff
@@ -80,10 +81,36 @@ func (t *Token) SigningString() (string, error) {
 	return strings.Join(parts, "."), nil
 }
 
+func Parse(tokenString string, keyFunc Keyfunc) (*Token, error) {
+	claim := make(MapClaim)
+	return ParseInterface(tokenString, keyFunc, &claim)
+}
+
+type MapClaim map[string]interface{}
+
+func (m MapClaim) ExpiresAt() (int64, error) {
+	if exp, ok := m["exp"].(float64); ok {
+		return int64(exp), nil
+	}
+	return 0, errors.New("expiration not found")
```

Comment:
If `exp` exists and is not a number, it shouldn't be skipped.  Validation should fail with "invalid value for exp".  Is `return 0, nil` a better option?  0 isn't a valid value for either `exp` or `nbf`.